### PR TITLE
Build a Snowflake Native App to Analyze Chairlift Sensor Data

### PR DESCRIPTION
**Pull request by sfc-gh-ghernandez**

### CODE REVIEW

File: site/sfguides/src/native-app-chairlift/native-app-chairlift.md [hunk 1]
```diff
@@ -0,0 +1,641 @@
...
+# Build a Snowflake Native App to Analyze Chairlift Sensor Data
+<!-- ------------------------ -->
+## Introduction
+Duration: 1
+
+In this Quickstart, you'll build a Snowflake Native Application that can analyze sensor data from chairlifts at different locations at a ski resort. Let's explore the scenario from the perspective of the application provider and an application consumer.
+
+– **Provider** – The provider of the app is a chairlift manufacturer. The app, once installed by a consumer, can run against chairlift sensor data housed in the consumer's account, and provide the consumer with important analytics and insights on the condition of each chairlift. 
+
+– **Consumer** – The consumer of the app is a customer of the chairlift manufacturer. They have purchased and installed chairlifts in various locations across a ski resort they own, and they collect raw sensor data from the chairlifts using a connector that ingests this data directly into their account. The provider's app runs against this data and helps the customer identify chairlifts in need of maintenance.
```
Comment (sfc-gh-cgorrie) [inline site/sfguides/src/native-app-chairlift/native-app-chairlift.md:19]: Great wording, this makes it all very clear!

File: site/sfguides/src/native-app-chairlift/native-app-chairlift.md [hunk 2]
```diff
@@ -0,0 +1,641 @@
...
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Data Science, Data Engineering, Apps 
+
+# Build a Snowflake Native App to Analyze Chairlift Sensor Data
+<!-- ------------------------ -->
+## Introduction
+Duration: 1
+
+In this Quickstart, you'll build a Snowflake Native Application that can analyze sensor data from chairlifts at different locations at a ski resort. Let's explore the scenario from the perspective of the application provider and an application consumer.
```
Comment (sfc-gh-cgorrie) [inline site/sfguides/src/native-app-chairlift/native-app-chairlift.md:15]: Perhaps you can mention that this quickstart is limited to a single-account installation, and that things like listing / versions / release directives are outside of the scope.

File: site/sfguides/src/native-app-chairlift/native-app-chairlift.md [hunk 3]
```diff
@@ -0,0 +1,641 @@
...
+
+![Sensor data](assets/sensor-data.png)
+
+
+### What You’ll Learn 
+
+- How to create an application package
+- How to create grants in the application package to control access to different parts of the app, depending on Snowflake role 
+- How to create a new version of the app
+- How to install and run the app in a Snowflake account
```
Comment (sfc-gh-cgorrie) [inline site/sfguides/src/native-app-chairlift/native-app-chairlift.md]: Perhaps "How to install and run the app as a different role in the same Snowflake account"? Maybe I'm being too pedantic.

File: site/sfguides/src/native-app-chairlift/native-app-chairlift.md [hunk 4]
```diff
@@ -0,0 +1,641 @@
...
+
+<!-- ------------------------ -->
+## Prepare objects in account
+Duration: 4
+
+Next, you'll run some scripts to setup some databases, schemas, and tables needed by the app.
+
+The scripts will do a couple of things:
+
+* The provider scripts define roles, warehouses, and data types for the data emitted by the sensors (i.e., brake temperature, motor RPMs, etc.). This is necessary so that the app can function as intended in the consumer's account. From this perspective, the provider is the chairlift manufacturer, and the app will run against raw sensor data in (and owned by) a consumer's account. Running the app against this data will provide the consumer with analytics and insights on the chairlifts they own, granted that the consumer installs the app and grants the proper privileges to the app.
```
Comment (sfc-gh-cgorrie) [inline site/sfguides/src/native-app-chairlift/native-app-chairlift.md]: Could mention that this data is combined with the consumer data inside of the app.

**Review by sfc-gh-cgorrie — APPROVED** [review]
A few optional nits, overall looks good to me!

File: site/sfguides/src/native-app-chairlift/native-app-chairlift.md (hunk 3 above)
Comment (sfc-gh-ghernandez) [reply site/sfguides/src/native-app-chairlift/native-app-chairlift.md]: I think that's a good callout. I went with:

> How to install and run the app in a single Snowflake account using multiple roles

**Comment by sfc-gh-ghernandez** [comment]
> A few optional nits, overall looks good to me!

Thank you! Updated.

**Pull request merged**